server/base/src/main/java/org/apache/accumulo/server/security/AuditedSecurityOperation.java

```diff
@@ -286,10 +286,10 @@ public boolean canDeleteTable(TCredentials c, TableId tableId, NamespaceId names
     String tableName = getTableName(tableId);
     try {
       boolean result = super.canDeleteTable(c, tableId, namespaceId);
-      audit(c, result, CAN_DELETE_TABLE_AUDIT_TEMPLATE, tableName, tableId);
+      audit(c, result, CAN_DELETE_TABLE_AUDIT_TEMPLATE, tableName);
```

Comment:
Why drop the tableId here?

Reply:
`CAN_DELETE_TABLE_AUDIT_TEMPLATE` was not set up to print it.  I could leave and change the template to print it.

Reply:
I think it's better to print it. It's much more useful for security/auditing purposes than a tableName (which could represent different tables over time), if it's available.

Reply:
I tried to print out the ids but it breaks AuditMessageIT which runs tests and verifies the audit messages. I tried to fix AuditMessageIT but had trouble.  I think this is why ids were left out of audit message. I think this is a larger issue that should be done in separate issue.

Reply:
If it needs to be done separately, then I think it's best if the parameter continues to be passed for now... no sense in changing it, only to change it back when the follow-on issue is addressed.

Reply:
Adding back the parameter will cause the warning again which I would like to avoid.

Reply:
The warning isn't a compiler warning, or any other kind of warning outside of LGTM.com. Is that correct?

I'm a big fan of removing warnings (some might say obsessed), but only because I see warnings as untriaged potential bugs. Removing them means that you've triaged and confirmed that there isn't a bug. In this case, though, the warning is doing it's job, highlighting an actual problem (lack of sufficient auditing, in this case). In that case, I think it's best to leave the warning until the problem is actually fixed.

Reply:
It's just a LGTM warning. I added the parameters back in. See b059812c3da74e57